common/src/main/java/com/wynntils/commands/ServerCommand.java

```diff
@@ -82,8 +82,7 @@ private int serverInfo(CommandContext<CommandSourceStack> context) {
         if (!ServerListModel.forceUpdate(UPDATE_TIME_OUT_MS)) {
             context.getSource()
                     .sendFailure(
-                            new TextComponent("Failed to get server data from API.").withStyle(ChatFormatting.RED));
-            return 1;
+                            new TextComponent("Network problems; using cached data").withStyle(ChatFormatting.RED));
```

Comment:
I am pretty sure this works, but it is a bit weird that we send a failure, but then, send a "success" as well.

Reply:
Yes, it's kind of weird. Sounds almost as weird to call `sendSuccess` as well. I looked at the implementation, both methods do more or less the same. 

This was not really part of the real fix, just something I noticed while rummaging about in the ServerCommand source and thought was an, unrelated, improvement. I can leave it out for now as well, too, and we can revisit this separately and see if there is some better way (maybe a third `sendUpdate` method?) to do this.

Reply:
This is fine. No need to revert it.